lib/tarpon/request/subscriber/entitlement.rb

```diff
@@ -4,8 +4,8 @@ module Tarpon
   module Request
     class Subscriber
       class Entitlement < Base
-        def initialize(subscriber_path:, entitlement_identifier:)
-          super()
+        def initialize(subscriber_path:, entitlement_identifier:, **opts)
```

Comment:
Same as my previous suggestion to explicitly require a client to be passed:
```suggestion
        def initialize(subscriber_path:, entitlement_identifier:, client:)
```